frontend/src/components/CollapseButton.tsx

```diff
@@ -36,24 +37,27 @@ const DefaultText = () => {
 const CollapseButton = ({
   children = <DefaultText />,
   expanded,
+  withoutBadge,
   onClick,
 }: CollapseButtonProps) => {
   const classes = useStyles();
   return (
-    <Button
-      color="secondary"
-      size="large"
-      startIcon={!expanded ? <ExpandMore /> : <ExpandLess />}
-      aria-expanded={expanded}
-      aria-label="show more"
-      onClick={onClick}
-      className={clsx(classes.collapseButton, {
-        [classes.collapseButtonDefault]: !expanded,
-        [classes.collapseButtonExpanded]: expanded,
-      })}
-    >
-      {children}
-    </Button>
+    <Badge color="secondary" variant="dot" invisible={withoutBadge}>
```

Comment:
I find it looks better with `collapseButton: { padding: '0px 8px', ... }`, otherwise the badge appears very high about the button text.

![Screenshot 2022-02-13 at 08 29 19](https://user-images.githubusercontent.com/8818081/153743752-b22d1d6f-e028-4af7-afd9-141b570908e3.png)


![Screenshot 2022-02-13 at 08 29 37](https://user-images.githubusercontent.com/8818081/153743753-95e21523-a7d5-444f-a5ac-3057973fbd68.png)